packages/core/src/conflict-resolution.ts

```diff
@@ -290,9 +306,8 @@ export class ConflictResolution {
     // `fetchLog` provides the timestamps.
     if (state && state.log.length === 1) {
       const timestamp = unappliedCommits[0].timestamp
-      const genesisCid = state.log[0].cid
-      const genesis = await this.dispatcher.retrieveCommit(genesisCid)
-      await handler.applyCommit(genesis, { cid: genesisCid, timestamp: timestamp }, this.context)
+      const genesis = await getCommitData(state.log[0], this.dispatcher)
+      await handler.applyCommit(genesis, { cid: genesis.cid, timestamp: timestamp }, this.context)
```

Comment:
I'm confused now the relationship between `CommitData` and `CommitMeta`.  They both seem to have a `cid` and `timestamp`.  Do we need both?

Reply:
Removed `CommitMeta`.